editor/src/clj/editor/game_project_core.clj

```diff
@@ -24,7 +24,7 @@
 
 (def meta-settings (:settings basic-meta-info))
 
-(defn default-settings []
+(def default-settings
```

Comment:
Changed this from a function to a constant since it will produce the same value every time.